### PR TITLE
Add supporting filter of the PSR-2

### PR DESCRIPTION
**Pull request by almost-online**
Add supporting PSR-2 method namign
Replace satooshi/php-coveralls to php-coveralls/php-coveralls

- [X] I've run the tests with `vendor/bin/phpunit`
- [X] None of the tests were found failing
- [X] I've seen the coverage report at `build/coverage/index.html`
- [X] Not a single line left uncovered by tests
- [X] Any coding standards issues were fixed with `vendor/bin/php-cs-fixer fix`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/
 vendor/
 composer.lock
+.php_cs.cache
```
Comment (sanmai) [inline .gitignore]: This must go into global .gitignore, please.

**Comment by sanmai** [comment]
This is quite a lot of changes. Would you be willing to provide some sort of an extended rationale for this PR?

**Comment by almost-online** [comment]
> 
> 
> This is quite a lot of changes. Would you be willing to provide some sort of an extended rationale for this PR?

One main thing is: our project have strict PSR-2 standard naming and we need add some extra filter.
e.g.  

```php
class SomeFilter {

public function myFilter($val): string
{
   return 'some' . $val;
}

}
```
and use this filter
```text
{{  value | my_filter }}
```

other case: we need extend exists filter 'remove_first' (for example)
```php
class ExtendStandardFilter 
{
public static function removeFirst($input, $string) 
{
   if (......) {
      return $string;
   }

  return StandardFilters::remove_first($input, $string);
}
}
```

**Comment by sanmai** [comment]
So it is like you want `{{  value | my_filter }}` to work even if you want a method to be named `myFilter`?

No offence, but how about if you just use `{{  value | myFilter }}` and be done with that?

**Comment by almost-online** [comment]
ok, I see and I close this PR cause this PR will broke some different filters but has the same name like one already exists filter like `remove_first` and another like  'removeFirst'. The last one will overwrite the previous one.

**Comment by sanmai** [comment]
Ow, that's unfortunate. Feel free to reopen though if you have something else in mind.

**Comment by almost-online** [comment]
Sure!
And I`d recommended to you to replace `satooshi/php-coveralls` to `php-coveralls/php-coveralls` cause:

```
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
Package satooshi/php-coveralls is abandoned, you should avoid using it. Use php-coveralls/php-coveralls instead.
```

**Comment by sanmai** [comment]
Yeah, I'm doing just that in #111

**Pull request closed without merging**